test/common/pixel-tests

```diff
@@ -81,11 +81,59 @@ cmd_reset() {
     cmd_pull
 }
 
+pixel_test_logs_url() {
+    arg=${1:-}
+
+    if [[ "$arg" == http* ]]; then
+        echo $arg
+        return
+    fi
+
+    repo=$(git remote get-url origin | sed -e 's,\(git@github.com:\|https://github.com/)\),,'  -e 's,\.git$,,')
+
+    if [ -n "$arg" ]; then
+        revision=$(curl -s https://api.github.com/repos/$repo/pulls/"$arg" | python -c "
```

Comment:
Please quote `$repo` as well, or just quote the entire URL. Please also call `python3` explicitly, this won't work with py2.

`... | jq -r .head.sha` would be perfect for this, it's just unfortunately less reliably installed than Python.

Reply:
Done. As discussed, the answer is always "more python" and never "less python".  So let's rewrite this whole script in Python eventually.